drivers/usb/device/usb_dc_sam0.c

```diff
@@ -132,6 +132,9 @@ static void usb_sam0_load_padcal(void)
 	u32_t pad_transp;
 	u32_t pad_trim;
 
+#ifdef USB_FUSES_TRANSN_ADDR
+	pad_transn = *(uint32_t *)USB_FUSES_TRANSN_ADDR;
```

Comment:
This change is not explained in the commit message. Where is USB_FUSES_TRANSN_ADDR defined, HAL?

Reply:
Yes, it's in the [`component/nvmctrl.h`](https://github.com/zephyrproject-rtos/hal_atmel/blob/master/asf/sam0/include/same54/component/nvmctrl.h#L743) header file.